crd-generator/src/main/java/io/strimzi/crdgenerator/DefaultValueSelector.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.crdgenerator;
+
+public class DefaultValueSelector {
```

Comment:
Err, this shouldn't have been committed. Will push a change to PR to remove it tomorrow.